library/tasks/Attach10xBarcodes.wdl

```diff
@@ -13,19 +13,19 @@ task Attach10xBarcodes {
   Int preemptible = 0
 
   meta {
-    description: "attaches barcodes found in r1 and i1"
+    description: "attaches barcodes found in r1 (forward) and i1 (index) fastq files to corresponding reads in the r2 (reverse) bam file"
   }
 
   parameter_meta {
     r1_fastq: "forward fastq file; contains umi, cell barcode"
     i1_fastq: "optional, index fastq file; contains sample barcode"
-    r2_unmapped_bam: "reverse bam file; contains alignable genomic information"
+    r2_unmapped_bam: "reverse unmapped bam file; contains alignable genomic information"
     whitelist: "10x genomics cell barcode whitelist for 10x V2"
-    docker: "optionally provide a docker image"
-    machine_mem_mb: "optionally provide how much memory(MB) to provision"
-    cpu: "optionally provide how many cpus to provision"
-    disk: "optionally provide how much disk to provision"
-    preemptible: "optionally provide how many preemptible attempts"
+    docker: "(optional) the docker image containing the runtime environment for this task"
```

Comment:
hmm i tried to stay away from putting any kind of structure in these descriptions since trying to add a structure to a string seems kinda wrong and error prone.  The defaults are interesting, I looked at these inputs as "power user" inputs who would have to look at the task itself to know how these values are being used before assigning values themselves.  It really depends if we want these this `parameter_meta` block to be used as simple descriptions or something more complicated.

Reply:
We could remove the defaults. They are listed above. I prefer the way I wrote the strings (obviously, heh) that follow, which don't have much structure to them.

Reply:
i just wish we could do like
```
parameter_meta {
  input_name: "description", isOptional
}
```
so we could do
```
parameter_meta {
  docker: "the docker image containing the runtime environment for this task", true
}
```